src/sass/components/govcy-header.scss

```diff
@@ -4,6 +4,7 @@
 //$govcy-primary: #31576F;
 $govcy-link-color: #254355;
 $govcy-sub-menu-color:#577589;
```

Comment:
Quite like having this variable here `govcy-sub-menu-color` because we're only going to need it in this file

Reply:
The one above, I know that's been there for a while so not necessarily part of this work but could it be moved to the vars file? I think it predates the vars file.

Reply:
Can we remove the line above?

Reply:
it will be fixed

Reply:
👍